.vscode/tasks.json

```diff
@@ -1,25 +1,75 @@
 {
-  "version": "2.0.0",
-  "tasks": [
-    {
-        "label": "Start React client",
-        "command": "cd ./frontend;npm start;cd ..;",
-        "args": [],
-        "type": "shell",
-        "problemMatcher": [],
-        "group": {
-            "kind": "build"
-        }
-    },
-    {
-        "label": "Install React Node packages",
-        "command": "cd ./frontend;npm install;cd ..;",
-        "args": [],
-        "type": "shell",
-        "problemMatcher": [],
-        "group": {
-            "kind": "build"
-        }
-    }
-]
-}
\ No newline at end of file
+	"version": "2.0.0",
```

Comment:
@glaizacano Is it documented on how we can run this locally? I haven't done it before on my end and am just curious :)

Reply:
Not yet, but I'll update the documentation soon with instructions on how to run the VS Code tasks locally.